test/xpu/xpu_test_utils.py

```diff
@@ -514,28 +515,30 @@ def replace_opinfo_device(self, info: OpInfo):
             info.decorators = tuple(decorator_xpu)
         skip_xpu = []
         replaced = False
-        for skip in info.skips:
-            if type(skip) == DecorateInfo:
-                if skip.device_type == "cuda":
-                    skip_xpu.append(decorator)
-                    skip.device_type = "xpu"
+        if hasattr(info, "skips"):
```

Comment:
You are saying there is no `skips` in some `onInfo`?

Reply:
Class `ModuleInfo` has the number `skips`. `hasattr` is not necessary.

Reply:
There is no skips in ModuleInfo.